cmiles/tests/reference/generate_reference.py

```diff
@@ -5,7 +5,7 @@
 # Regenerate with molecule names (makes it easier to search later)
 
 # Make sure to add SMILES to input smi file. RDKit expects that line there and just skips the first line
-mols = rdkit.Chem.SmilesMolSupplier('drug_bank_stereo.smi')
+mols = rdkit.Chem.SmilesMolSupplier('drug_bank_stereo.smi', titleLine=False)
```

Comment:
(not blocking) did the previous results file not have this title line by default? I'm wondering if RDKit just recently changed this behavior?

Reply:
Just checked with separate environments and it's not an RDKit change. I think this is something that was done by hand while running this script again - which I found frustrating, hence the change here.